src/state/state.ts

```diff
@@ -19,16 +11,16 @@ interface GasPriceInfo {
 
 export interface State {
   config: Config;
-  gasPrices: Record<ChainId, Record<ProviderName, GasPriceInfo[]>>;
+  gasPrices: Record<string /* Chain ID */, Record<string /* Provider name */, GasPriceInfo[]>>;
```

Comment:
Can we use [this](https://github.com/api3dao/commons/blob/fb07e2c083cf63bc35d500aca37eb3fb7ee411c8/src/blockchain-utilities/schema.ts#L33) for `chainId`?